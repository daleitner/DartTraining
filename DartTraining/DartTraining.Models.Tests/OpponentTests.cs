using ApprovalTests;
using ApprovalTests.Reporters;
using Base;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace DartTraining.Models.Tests
{
	[TestClass]
	[UseReporter(typeof(DiffReporter))]
	public class OpponentTests
	{
		protected Opponent SUT;

		[TestInitialize]
		public void Setup()
		{
			IdGenerator.PrepareGeneratorForTest(0);
		}

		[TestMethod]
		public void WhenCreateOpponent_ThenVerifySUT()
		{
			SUT = new Opponent("some","player");
			Approvals.Verify(SUT.ToPrettyString());
		}

		[TestMethod]
		public void VerifyToString()
		{
			SUT = new Opponent("some", "player");
			Approvals.Verify(SUT.ToString());
		}
	}
}
