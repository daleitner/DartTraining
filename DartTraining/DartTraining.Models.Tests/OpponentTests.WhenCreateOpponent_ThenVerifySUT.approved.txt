Δ Opponent
┌──────────────────────────────────────┬───────────────┬────────────┬────────┐
│ [Id]                                 │ [DisplayName] │ [LastName] │ [Name] │
├──────────────────────────────────────┼───────────────┼────────────┼────────┤
│ 00000000-0000-0000-0000-000000000000 │               │ player     │ some   │
└──────────────────────────────────────┴───────────────┴────────────┴────────┘
